### ISSUE TITLE
Using rtl_tcp

### ISSUE THREAD

**Issue by g7ruh** [jvde-github/AIS-catcher-for-Android#12]
Jasper.

I have been testing all the interfaces but I cannot get rtl_tcp to work.

Can you give me a few points as to setup on the Pi and phone, not sure I have done it right or if the is a problem. Most likely the former!

Roger

**Comment by g7ruh** [comment]
I think it was finger trouble, It started when I used the settings (adjusted for my network) you used.

It is working but only to about 86% of the live system. I suspect that is because you are forcing auto gain. Is it possible to either allow a gain setting or stop forcing it and allow rtl_tcp to set gain as in -g 19.2

**Comment by g7ruh** [comment]
I have just tried RTLTCP with manual gain settings from 1 to 50 in steps of 5 or 10. All result in a message rate of less than 1.0. Selecting Auto results in a message rate of 10.1, compared with 10.5 on the live system.

e.g.

set freq correction 0
set gain mode 1
set gain 5
set sample rate 240000
set freq 162000000
comm recv bye

and 
Allocating 15 zero-copy buffers
set freq correction 0
set gain mode 0
set sample rate 240000
set freq 162000000

These are using RTL-SDR dongle V3 so PPM is 0

RTLSDR works nicely and I get a rate of 10.9 compared with 11.3 on the live system with a gain of 28/29

**Comment by g7ruh** [comment]
Jasper, all fixed and I did a bit of comparing, sample rate wise.

All using RTL-SDR V3 SN444 to test. The live system uses RTL-SDR V3 SN555. I agree that the higher sample rate of 1536 is better and offers comparable performance to AIS-catcher on the Pi.

You will notice that the gain is near maximum for the dongle. I have used the FA Orange as a check and the 'sweet spot' for that dongle is a gain of about 24.

Using sample rate of 2304 is similar to 1536. 

![image](https://github.com/jvde-github/AIS-catcher-for-Android/assets/5949609/1339dc10-ad48-451a-a7fc-4f782da455af)


Hope that helps, thanks for fixing the issue.
Roger


PS, yesterday afternoon was a busy one on the water:

![image](https://github.com/jvde-github/AIS-catcher-for-Android/assets/5949609/8fd126d2-dc7c-4ba9-a141-13b8a1f4db5f)

![image](https://github.com/jvde-github/AIS-catcher-for-Android/assets/5949609/f1135fdf-8c24-4c48-98ba-e48fddaf7dc0)
